packages/tck-tests/src/tests/create.test.ts

```diff
@@ -0,0 +1,47 @@
+import { TestClient } from '../TestClient'
+
+describe('create-tck', () => {
+  const schema = /* Prisma */ `
+    generator client {
+      provider = "prisma-client-js"
+    }
+    
+    datasource db {
+      provider = "postgresql"
+      url      = "DATABASE_URI"
+    }
+    
+    model User {
+      id String @id
+      username  String
+    }
+  `
+
+  const client = new TestClient({
+    schema,
+    provider: 'postgresql',
+    uri: process.env.TEST_POSTGRES_URI as string,
+  })
+
+  beforeAll(async () => {
+    await client.generate()
+  })
+
+  test('simple create', async () => {
+    const query = () => client.prisma?.user.create({ data: { id: 'some-random-id', username: 'some-random-username' } })
```

Comment:
What do you think about passing `prisma` as the argument of the `client.query` callback so that it is self-contained and doesn't need to depend on the same instance of `TestClient` via a closure?

Reply:
You can also do the null check inside `client.query` itself if it's necessary, so that the API will look like this basically:

```ts
    const query = (prisma) => prisma.user.create({ data: { id: 'some-random-id', username: 'some-random-username' } })

    const response = await client.query(query)
```